src/app/grid/grid-boston-marathon/grid.component.scss

```diff
@@ -1,135 +1,114 @@
 @import '~igniteui-angular/lib/core/styles/themes/index';
-
-.default-theme {
-    $grid-sample-theme: igx-grid-theme(
-        $row-selected-background: #333,
-        $row-selected-text-color: #ddd,
-        $row-hover-background: #f8f8f8,
-        $row-border-color: #f8f8f8,
-
-        $cell-selected-background: igx-color($default-palette, grays, 800),
-        $cell-selected-text-color: #fff
-      );
-
-      $progressBar-sample-theme: igx-progress-linear-theme(
-        $track-color: rgba(181,181,181, .5),
-        $fill-color-default: orange
-      );
-
-      .grid__wrapper {
+:host {
+     ::ng-deep {
+        $grid-sample-theme: igx-grid-theme( $row-selected-background: #333, $row-selected-text-color: #ddd, $row-hover-background: #f8f8f8, $row-border-color: #f8f8f8, $cell-selected-background: igx-color($default-palette, grays, 800), $cell-selected-text-color: #fff);
```

Comment:
I think this theme should be applied to the when the body has a `dark-theme`, with the configuration here, the Bost grid cannot be themed
![bstgrid](https://user-images.githubusercontent.com/25771040/62599371-b59ce480-b8f4-11e9-9130-57ff19363adb.PNG)